packages/corvid-cli/src/commands/pull.js

```diff
@@ -40,7 +40,7 @@ async function pullCommand(spinner, args) {
         },
         error: error => {
           spinner.fail();
-          const errorMessage = getMessage(error);
+          const errorMessage = getMessage(error.message);
           if (errorMessage) {
             if (error === "CAN_NOT_PULL_NON_EMPTY_SITE") {
```

Comment:
I think you may be able to remove this if, since a UserError is already thrown from the server, and the only difference I can see is coloring, which you handle in a different place

Reply:
not sure... it changes the error message... we'll need to refactor this whole thing soon